pkg/imagestore/imagestore_test.go

```diff
@@ -175,23 +175,100 @@ var _ = Describe("HaveVersion", func() {
 		store ImageStore
 	)
 
-	BeforeEach(func() {
-		var err error
-		store, err = NewImageStore(nil, "", versions)
-		Expect(err).NotTo(HaveOccurred())
-	})
-	AfterEach(func() {
-	})
+	Describe("HaveVersion", func() {
+
+		BeforeEach(func() {
+			var err error
+			store, err = NewImageStore(nil, "", versions)
+			Expect(err).NotTo(HaveOccurred())
+		})
+		AfterEach(func() {
+		})
+
+		It("is true for versions that are present", func() {
+			Expect(store.HaveVersion("4.8", "x86_64")).To(BeTrue())
+			Expect(store.HaveVersion("4.9", "arm64")).To(BeTrue())
+		})
 
-	It("is true for versions that are present", func() {
-		Expect(store.HaveVersion("4.8", "x86_64")).To(BeTrue())
-		Expect(store.HaveVersion("4.9", "arm64")).To(BeTrue())
+		It("is false for versions that are missing", func() {
+			Expect(store.HaveVersion("4.9", "x86_64")).To(BeFalse())
+			Expect(store.HaveVersion("4.8", "arm64")).To(BeFalse())
+			Expect(store.HaveVersion("4.7", "x86_64")).To(BeFalse())
+			Expect(store.HaveVersion("4.8", "aarch64")).To(BeFalse())
+		})
 	})
 
-	It("is false for versions that are missing", func() {
-		Expect(store.HaveVersion("4.9", "x86_64")).To(BeFalse())
-		Expect(store.HaveVersion("4.8", "arm64")).To(BeFalse())
-		Expect(store.HaveVersion("4.7", "x86_64")).To(BeFalse())
-		Expect(store.HaveVersion("4.8", "aarch64")).To(BeFalse())
+	Describe("ValidateVersions", func() {
+		var (
+			ctrl       *gomock.Controller
+			mockEditor *isoeditor.MockEditor
+			dataDir    string
+		)
+
+		BeforeEach(func() {
+			var err error
+			dataDir, err = os.MkdirTemp("", "imageStoreTest")
```

Comment:
I don't think you need this directory either as we're not doing anything with it, but if you are going to create it then you need to clean it up in an `AfterEach` block.